add-node-modules-path.el

```diff
@@ -48,10 +48,10 @@ Traverse the directory structure up, until reaching the user's home directory,
  or hitting add-node-modules-max-depth.
 Any path found is added to the `exec-path'."
   (interactive)
-  (let* ((file (or (buffer-file-name) default-directory))
+  (let* ((file (or (buffer-file-name) (expand-file-name default-directory)))
```

Comment:
Can we refactor it so that `(expand-file-name default-directory)` is a let variable and used where needed?

Reply:
Yep, thanks for the feedback, I'll update my PR